src/visualizations/config/adapters/dhis_highcharts/addTrendLines.js

```diff
@@ -77,10 +73,11 @@ function getRegressionObj(data, regressionType) {
     // - decimalPlaces (default = 2)
 
     let regression
-    let regressionTypeOptions = {}
+    const regressionTypeOptions = {}
 
     switch (regressionType) {
         case 'LINEAR':
```

Comment:
I don't think I've seen `default` other than last in a switch. Could it be moved to the bottom, just for convention's sake? Actually, I am a little nervous about some of these functional changes (this one, and the == to === changes) and any effects they might have in the apps. I would almost prefer to make individual eslint exceptions (output warning instead of error), and then address them individually.

Reply:
This is a functional change, true, but the current behavior in the default case (which is hopefully never hit) is to cause an access violation (cannot read property x of undefined) a few lines later, so this is definitely a safe improvement I think.

Which == to === changes do you have issues with?  I think they were mostly incorrect before this change.

Reply:
I was just thinking generally about risk of going to a stricter equality check. Might mean that other functions using this were being lazy about types, and suddenly those functions get unexpected results.

Reply:
Rearranged the defaults

Reply:
Discussed with @edoardo and in slack - the `=== undefined` changes should be OK